Source/Timeline/TimelinePagerView.swift

```diff
@@ -408,10 +408,14 @@ public final class TimelinePagerView: UIView, UIGestureRecognizerDelegate, UIScr
     delegate?.timelinePager(timelinePager: self, willMoveTo: newDate)
 
     func completionHandler(_ completion: Bool) {
-      DispatchQueue.main.async {
+      DispatchQueue.main.async { [self] in
```

Comment:
Should a `weak self` be used here?

Reply:
I don't think so. When I change it to ```weak self``` it making ```self``` optional